scripts/push-bundle-size.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Function to log messages
+log() {
+    echo "[$(date +'%Y-%m-%d %H:%M:%S')] $1"
+}
+
+# Check if running in GitHub Actions
+if [[ "${GITHUB_ACTIONS:-}" != 'true' ]]; then
+    log "Error: This script must be run in a GitHub Actions environment"
+    exit 1
+fi
+
+if [[ -z "${GITHUB_ACTOR:-}" ]]; then
+    log "Error: GITHUB_ACTOR environment variable must be set"
+    exit 1
+fi
+
+# Check for MOBILE_BUNDLESIZE_TOKEN
+if [[ -z "${MOBILE_BUNDLESIZE_TOKEN:-}" ]]; then
+    log "Error: MOBILE_BUNDLESIZE_TOKEN environment variable must be set"
+    exit 1
+fi
+
+# Set up git configuration
+git config --global user.email "metamaskbot@users.noreply.github.com"
+git config --global user.name "MetaMask Bot"
+
+# Create a temporary directory
+temp_dir=$(mktemp -d)
```

Comment:
```suggestion
temp_dir="$(mktemp -d)"
```
Quote-wrap `mktemp` output for safety.